src/main/java/edu/cornell/kfs/concur/batch/ConcurRequestExtractCreatePdpImportStep.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.kfs.concur.batch;
+
+import java.util.Date;
+
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpImportService;
+
+public class ConcurRequestExtractCreatePdpImportStep extends AbstractStep {
+	
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpImportStep.class);
```

Comment:
I think this file needs reformatting. Last 2 methods seems to have different formatting than the rest of the file.